addons/mail/static/src/discuss/call/common/call_participant_card.xml

```diff
@@ -34,7 +33,10 @@
                         LIVE
                     </small>
                 </span>
-                <div class="o-discuss-CallParticipantCard-overlay position-absolute top-0 end-0 d-flex flex-row-reverse">
+                <button t-if="isContextMenuAvailable and ((!isMobileOS and rootHover.isHover) or (isMobileOS and !props.minimized))" t-on-click="onContextMenu" class="o-discuss-CallParticipantCard-contextButton btn btn-secondary btn-sm rounded-circle position-absolute border-0 top-0 end-0">
+                    <i class="fa fa-chevron-down" />
+                </button>
+                <div class="o-discuss-CallParticipantCard-overlay position-absolute bottom-0 end-0 d-flex flex-row-reverse">
```

Comment:
Anchor of call context menu should be the chevron rather than the card:
<img width="965" alt="Screenshot 2025-01-16 at 16 20 06 (2)" src="https://github.com/user-attachments/assets/dc67d95f-aba6-441c-b140-dc7f9ae8b20a" />